covid19.occ-pla.net/manifests/modsec/iis-errors.data

```diff
@@ -0,0 +1,13 @@
+<h2 style="font:8pt/11pt verdana; color:000000">HTTP 403.6 - Forbidden: IP address rejected<br>
```

Comment:
i think WAF is now deployed automatically, so you don't need the `modsec` folder